cardano-node/test/Test/Cardano/Tracing/OrphanInstances/HardFork.hs

```diff
@@ -113,14 +113,14 @@ ntc_HFV1_ByronV1_ShelleyV6_ConwayV2 ::
     (Consensus.Cardano.CardanoEras StandardCrypto)
 ntc_HFV1_ByronV1_ShelleyV6_ConwayV2 =
```

Comment:
Same here

Reply:
Thanks for catching all of these renaming left-overs :raised_hands: !